blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineFolderImpl.java

```diff
@@ -97,12 +99,7 @@ public Integer getNumberOfPipelines() {
 
     @Override
     public BlueFavorite favorite(@JsonBody BlueFavoriteAction favoriteAction) {
-        if(favoriteAction == null) {
-            throw new ServiceException.BadRequestExpception("Must provide pipeline name");
-        }
-
-        FavoriteUtil.favoriteJob(folder.getFullName(), favoriteAction.isFavorite());
-        return FavoriteUtil.getFavorite(folder.getFullName(), this);
+        throw new ServiceException.MethodNotAllowedException("Cannot favorite a folder");
```

Comment:
We don't have a way to show that. When favouriting a MBP you are actually favouriting its master branch. The API reflects that now.